src/Orchard.Cms.Web/Startup.cs

```diff
@@ -36,6 +36,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddCommands();
 
+            services.AddAuthentication();
```

Comment:
Has this sth to do with the fact that in your original PR you skip changes in oauth branch for Orchard.Users/Startup.cs file? 
Those changes set dataProtectionProvider in IdentityOptions:
https://github.com/OrchardCMS/Orchard2/blob/oauth2/src/Orchard.Cms.Web/Modules/Orchard.Users/Startup.cs#L92
```
options.Cookies.ApplicationCookie.DataProtectionProvider = _dataProtectionProvider;
                options.Cookies.ExternalCookie.DataProtectionProvider = _dataProtectionProvider;
                options.Cookies.TwoFactorRememberMeCookie.DataProtectionProvider = _dataProtectionProvider;
                options.Cookies.TwoFactorUserIdCookie.DataProtectionProvider = _dataProtectionProvider;   
```
Should we undo those changes from oauth branch?